pages/common/conda.md

```diff
@@ -5,15 +5,15 @@
 
 - Create a new environment, installing named packages into it:
 
-`conda create --name {{environment_name}} {{python=2.7 matplotlib}}`
+`conda create --name {{environment_name}} {{python=3.7 matplotlib}}`
 
 - List all environments:
 
 `conda info --envs`
 
 - Load or unload an environment:
 
-`conda {{activate|deactivate}} {{environment_name}}`
+`conda {{activate environment_name|deactivate}}`
```

Comment:
This looks a bit strange, but I'd say it's OK. 👍🏻

Reply:
I fear it might confuse some. Would be great if we could split that up into two examples. We'd have to remove another though

Reply:
@hervenivon we should remove another example to split this in 2 examples. I think we can remove the `create` example because we have `conde-create` in another page, or the `search` one or the `clean` based on the less used one. What do you think?

Reply:
+1 for removing the `create` example.

Reply:
If we absolutely need to remove one, I’ll go for the search which is not essential to the basic flow of using `conda`

Reply:
Done